src/i18n/messages/en.json

```
@@ -25,6 +25,14 @@
     "testimonials-date": "Reviewed on {date}",
     "testimonials-subtitle": "Testimonials",
     "testimonials-heading": "Real Words from Happy Customers",
+    "login": {
+        "title": "Welcome back!",
+        "noAccount": "Don't have an account yet?",
+        "createNow": "Create one now!",
+        "forgotpss": "Forgot your password?",
+        "login": "Login",
```

Biome (2.1.2)

[error] The key login was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)